src/component/mxgraph/MxGraphConfigurator.ts

```diff
@@ -68,6 +68,8 @@ export default class MxGraphConfigurator {
     if (mouseNavigationSupport) {
       this.graph.panningHandler.useLeftButtonForPanning = true;
       this.graph.panningHandler.ignoreCell = true; // ok here as we cannot select cells
+      this.graph.panningHandler.addListener(mxEvent.PAN_START, this.getPanningHandlerCallback('grab').bind(this.graph));
```

Comment:
To avoid some duplications, is it possible to have something like: 
```
      this.graph.panningHandler.addListener(mxEvent.PAN_START, this.getPanningHandler('grab'));
      this.graph.panningHandler.addListener(mxEvent.PAN_END, this.getPanningHandler('default'));
```
and 
```
  private getPanningHandler(cursor: 'grab' | 'default'): OmitThisParameter<(this: mxGraph) => void> {
    return this.getPanningHandlerCallback(cursor).bind(this.graph);
  }
```
?

Reply:
sure, good idea 👍